zigpy/ota/__init__.py

```diff
@@ -108,6 +109,8 @@ def __init__(self, app: ControllerApplicationType, *args, **kwargs):
             self.add_listener(zigpy.ota.provider.FileStore())
         if ota_config[CONF_OTA_IKEA]:
             self.add_listener(zigpy.ota.provider.Trådfri())
+        if ota_config[CONF_OTA_INOVELLI]:
+            self.add_listener(zigpy.ota.provider.Inovelli())
         if ota_config[CONF_OTA_LEDVANCE]:
             self.add_listener(zigpy.ota.provider.Ledvance())
         if ota_config[CONF_OTA_SALUS]:
```

Comment:
GitHub doesn't allow for review comments for distant lines of code but since your OTA JSON does not include the `image_type`, you will have to match OTA images based on a device's model name, like Salus currently does. You should be able to change the check to this:

```Python
if manufacturer_id in (
    zigpy.ota.provider.Salus.MANUFACTURER_ID,
    zigpy.ota.provider.Inovelli.MANUFACTURER_ID,
)
```